### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by HENRYKC24**
### In this pull request, I did the following:point_down:
1. Initialised React app.
2. Added React-Redux (npm install react-redux).
3. Structured my application files using a "feature folder" approach and use the 'ducks' pattern for my Redux files.
4. Set the building blocks of my app as re-usable components.
5. Added React Router and set two <Route>s and <Link>s for the app's navigation.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Book = ({ title, author, category }) => (
+  (
+    <ul>
+      <li>{title}</li>
+      <li>{author}</li>
+      <li>{category}</li>
+    </ul>
```
Comment (Davidosky007) [inline src/components/Book.js]: * Kindly add a button for removing books Note ✍️ the button wouldn't remove the books yet, this is just a placeholder for now 👍 
image

**Comment by Davidosky007** [comment]
## Hi @HENRYKC24 👋, 

Good job so far! 💪 
* Well written readme file ✔️ 
* Professional PR ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @HENRYKC24 👋, 

Good job so far! 💪 
* Well written readme file ✔️ 
* Professional PR ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Book.js (hunk 1 above)
Comment (HENRYKC24) [reply src/components/Book.js]: Oh!:smile: 
Done! Thank you, @Davidosky007

**Review by AshakaE — APPROVED** [review]
### Hi @HENRYKC24  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HENRYKC24** [comment]
Thank you @AshakaE:pray:

**Pull request merged**